superset/migrations/versions/2022-06-27_14-59_7fb8bca906d2_permalink_rename_filterstate.py

```diff
@@ -0,0 +1,91 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""permalink_rename_filterState
+
+Revision ID: 7fb8bca906d2
+Revises: f3afaf1f11f0
+Create Date: 2022-06-27 14:59:20.740380
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "7fb8bca906d2"
+down_revision = "f3afaf1f11f0"
+
+import pickle
+
+from alembic import op
+from sqlalchemy import Column, Integer, LargeBinary, String
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import Session
+
+from superset import db
+from superset.migrations.shared.utils import paginated_update
+
+Base = declarative_base()
+VALUE_MAX_SIZE = 2**24 - 1
```

Comment:
can we import this from the model file instead of defining it twice?

Reply:
We shouldn't import things from modules for migrations as modules may be move around and these constant values may change but migrations need to be deterministic.